packages/plugin/src/rules/require-import-fragment/index.ts

```diff
@@ -90,13 +90,16 @@ export const rule: GraphQLESLintRule = {
           const extractedImportPath = comment.value.match(/(["'])((?:\1|.)*?)\1/)?.[2];
           if (!extractedImportPath) continue;
 
-          const importPath = path.join(path.dirname(filePath), extractedImportPath);
+          const importPath = path.join(
+            path.dirname(filePath),
+            extractedImportPath.replaceAll(/[/\\]/g, path.sep),
+          );
+
           const hasInSiblings = fragmentsFromSiblings.some(
             source => source.filePath === importPath,
```

Comment:
previously you added `path.resolve(source.filePath)` but why?

`source.filePath` comes from `graphql-config` and contains slashes according to user's os

Reply:
I thought so too, but when I printed out the filePath in `fragmentsFromSiblings` by adding 
```
console.log(fragmentsFromSiblings.map(x=>x.filePath));
```

I saw this syntax. On Windows.

```
C:/Dev/my-project/client/logic/experiments/experimentFragment.graphql
```

Reply:
Strange because I added console log on ci and get backslash on windows

Reply:
here I commented https://github.com/dimaMachina/graphql-eslint/pull/2277/commits/d9f5b4ea2dd22999fdfa50452a689a7c6e341d8dand got 

![IMG_2887](https://github.com/user-attachments/assets/d1c562d9-1271-4197-a5bb-eaaaff32cc0a)

Reply:
I wonder maybe it happens when you specify glob syntax for `documents`? e.g. `**/*.gql`

Reply:
yeah, I was right, specifying glob pattern returns forward slash always

<img width="1096" alt="image" src="https://github.com/user-attachments/assets/76359656-dedc-4c3e-96b5-74c45783b528">